frontend/class-public.php

```diff
@@ -149,8 +149,10 @@ public static function register_post_types(){
 	 */
 	public function volunteer_opps_register_block(){
 
-		// Skip block registration if Gutenberg is not enabled
-		if ( ! function_exists( 'register_block_type' ) ) {
+		global $pagenow;
```

Comment:
This is to prevent the block script from being registered at all on the widgets.php page. If the script gets registered on the widgets.php page, it throws the following notice:

"PHP Notice:  wp_enqueue_script() was called <strong>incorrectly</strong>. "wp-editor" script should not be enqueued together with the new widgets editor (wp-edit-widgets or wp-customize-widgets)."

Reply:
Nice work sir. It's frustrating they didn't release an official way to remove a block from the Widgets screen.

Reply:
It really is. I have a feeling that'll come though.